Modules/_threadmodule.c

```diff
@@ -1527,8 +1544,8 @@ static PyMethodDef thread_methods[] = {
      METH_NOARGS, exit_doc},
     {"exit",                    thread_PyThread_exit_thread,
      METH_NOARGS, exit_doc},
-    {"interrupt_main",          thread_PyThread_interrupt_main,
-     METH_NOARGS, interrupt_doc},
+    {"interrupt_main",          (PyCFunction)thread_PyThread_interrupt_main,
+     METH_VARARGS, interrupt_doc},
```

Comment:
Suggestion (feel free to ignore): We could use `METH_O` instead

Reply:
Hmm... the parameter is optional.

Reply:
Oh, right, then nevermind